table/tables/memory_tables_test.go

```diff
@@ -92,10 +111,13 @@ func (ts *testMemoryTableSuite) TestMemoryBasic(c *C) {
 	c.Assert(row[0].GetInt64(), Equals, int64(1))
 
 	_, err = tb.AddRecord(ctx, types.MakeDatums(1, "aba"))
-	c.Assert(err, IsNil)
+	c.Assert(err, NotNil)
```

Comment:
The first column turns to be the primary key on line 68 and 70, so primary key conflicts here.